metadata-etl/src/main/java/metadata/etl/dataset/hdfs/HdfsMetadataEtl.java

```diff
@@ -95,20 +98,23 @@ private void extractLocal()
     String numOfThread = prop.getProperty(Constant.HDFS_NUM_OF_THREAD_KEY, String.valueOf(1));
     String hdfsUser = prop.getProperty(Constant.HDFS_REMOTE_USER_KEY);
     String hdfsKeyTab = prop.getProperty(Constant.HDFS_REMOTE_KEYTAB_LOCATION_KEY);
-
-    String execCmd =
-            "hadoop jar " + remoteJarFile
-            + " -D " + Constant.HDFS_SCHEMA_REMOTE_PATH_KEY + "=" + outputSchemaFile
-            + " -D " + Constant.HDFS_SAMPLE_REMOTE_PATH_KEY + "=" + outputSampleDataFile
-            + " -D " + Constant.HDFS_CLUSTER_KEY + "=" + cluster
-            + " -D " + Constant.HDFS_WHITE_LIST_KEY + "=" + whiteList
-            + " -D " + Constant.HDFS_NUM_OF_THREAD_KEY + "=" + numOfThread
-            + " -D " + Constant.HDFS_REMOTE_USER_KEY + "=" + hdfsUser
-            + " -D log_file_name=hdfs_schema_fetch" 
-            + " -D " + Constant.HDFS_REMOTE_KEYTAB_LOCATION_KEY + "=" + hdfsKeyTab;
-    //logger.info("executue remote command : " + execCmd);
-
-    Process process = Runtime.getRuntime().exec(execCmd);
+    String hdfsExtractLogFile = outputSchemaFile + ".log";
```

Comment:
This is good, separate the log for different job type makes it much easier for debugging, which were an headache before.